--- a/packages/SystemUI/src/com/android/systemui/qs/tileimpl/QSFactoryImpl.java
+++ b/packages/SystemUI/src/com/android/systemui/qs/tileimpl/QSFactoryImpl.java
@@ -45,7 +45,7 @@
     public QSTile createTile(String tileSpec) {
         if (tileSpec.equals("wifi")) return mWifiTile.get();
         if (tileSpec.equals("bt")) return mBluetoothTile.get();
-        if (tileSpec.equals("flashlight")) return mFlashlightTile.get();
+        if (tileSpec.equals("flashlight")) return mFlashlightTile.get();
         if (tileSpec.equals("settings")) return mSettingsTile.get();
         return null;
     }
